mobile_verifier/src/banning/db.rs

```diff
@@ -0,0 +1,105 @@
+use std::collections::HashSet;
+
+use chrono::{DateTime, Utc};
+use file_store::mobile_ban::{BanAction, BanType, VerifiedBanReport};
+use futures::TryStreamExt;
+use helium_crypto::PublicKeyBinary;
+use sqlx::{PgConnection, PgPool, Postgres, Row, Transaction};
+
+pub async fn get_banned_radios(
+    pool: &PgPool,
+    timestamp: DateTime<Utc>,
+) -> anyhow::Result<HashSet<PublicKeyBinary>> {
+    let banned = sqlx::query(
+        r#"
+            SELECT hotspot_pubkey
+            FROM hotspot_bans
+            WHERE 
+                ban_type != 'data'
```

Comment:
Maybe it is better to get specific bans?
In this case `ban_type == 'poc'` as I understand. 
In case new ban type appears here can be selected not only `poc` bans